WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2293,9 +2293,7 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
         boolean isFreeWPCom = mSite.isWPCom() && SiteUtils.onFreePlan(mSite);
         boolean isWPComSite = mSite.isWPCom() || mSite.isWPComAtomic();
 
-        boolean canViewEditorOnboarding = (
-                mAccountStore.getAccount().getUserId() % 100 >= (100 - EDITOR_ONBOARDING_PHASE_PERCENTAGE)
-                || BuildConfig.DEBUG) && !AppPrefs.hasLaunchedGutenbergEditor();
+        boolean canViewEditorOnboarding = canViewEditorOnboarding() && !AppPrefs.hasLaunchedGutenbergEditor();
```

Comment:
What would you think about renaming either this variable (and all the downstream variables) or the `canViewEditorOnboarding()` function so that they're different since they represent different things? Having them use the same name feels like something that will result in a bug sometime in the future to me. At a minimum, it feels confusing to me.

Reply:
I agree. I began to rename the capability itself within `GutenbergPropsBuilder` to be `editorOnboarding`, but paused to avoid needing to update Gutenberg itself. That said, if you also feel it is worthwhile, then I can make that change now.

Reply:
I think it is worthwhile. Perhaps you could just rename the function to something like `eligibleForEditorOnboarding` and avoid having to update Gutenberg.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/14766/commits/55d78c8d5bbe1d9be8ce0f10e7f9cf5af22d187c.